.github/workflows/reusable-build.yml

```diff
@@ -166,6 +166,21 @@ jobs:
             io.artifacthub.package.readme-url=https://raw.githubusercontent.com/${{ github.repository }}/main/README.md
             io.artifacthub.package.logo-url=https://avatars.githubusercontent.com/u/1728152?s=200&v=4
 
+      - name: Retrieve Signing Key
+        if: (github.event_name == 'scheduled' || github.event_name == 'workflow_dispatch' || github.event_name == 'merge_group') && github.event_name != 'pull_request'
```

Comment:
Don't think about it now, but it might be worth having a sample signing key that runs on PRs to make it easier to debug in future

Reply:
Oh, we have signing keys added in this PR anyway.  Any reason not to use the .test keys in PRs?

Reply:
We do, you can see in `fetch.sh` if the private-key is 0 bytes, we copy the test keys over.

Reply:
Ohhh, sorry.  We don't need to retrieve them since they are already in the repo :facepalm: